data_taking_scripts/motor_control.py

```diff
@@ -54,8 +54,8 @@ def inch_to_cm(dist):
             if prompt == 'o':
                 override = 1
 
-        print('Resonator length: {}'.format(current_resonator_length))
-        resonant_freq = logger.flmn(0,0,18,current_resonator_length)
+        print('Resonator length: {}'.format(current_resonator_length_cm))
+        resonant_freq = logger.flmn(0,0,18,current_resonator_length_cm)
```

Comment:
We are trying to move away from predicting the modes analytically. Can you have this print the value from the interpolation?